cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -276,6 +294,10 @@ pub mod pallet {
 		AlreadySuspended,
 		/// The execution is already resumed.
 		AlreadyResumed,
+		/// There are too many active outbound channels.
+		TooManyOutboundChannels,
```

Comment:
```suggestion
		TooManyActiveOutboundChannels,
```

Maybe?

Reply:
Is there a difference between open channels and active channels? Do open channels already have a limit?

Reply:
Open channels are not really tracked in this pallet. They are just provided by `T::ChannelInfo::get_channel_info(para_id)`, so the pallet has no clue how many channels are currently open.  
But it does track all "active" channels, so to say all channels that have messages queued.